include/meltpooldg/heat/heat_transfer_operator.hpp

```diff
@@ -135,14 +144,27 @@ namespace MeltPoolDG::Heat
     {
       AssertThrow(!level_set_as_heaviside || (velocity && level_set_as_heaviside),
                   ExcMessage("Two-phase flow must come with a velocity! Abort..."));
-      AssertThrow(material.first.conductivity > 0.0 && material.first.density > 0.0,
-                  ExcMessage(
-                    "The material's conductivity and density must be greater than zero! Abort..."));
+      AssertThrow(
+        (material.first.conductivity > 0.0 && material.first.density > 0.0) || data.solidification,
+        ExcMessage("The material's conductivity and density must be greater than zero! Abort..."));
       AssertThrow(
         !level_set_as_heaviside ||
           (material.second.conductivity > 0.0 && material.second.density > 0.0),
         ExcMessage(
           "The secondary material's conductivity and density must be greater than zero! Abort..."));
+      AssertThrow(
+        !data.solidification || (material.solid.conductivity > 0.0 && material.solid.density > 0.0),
```

Comment:
Would it be more precise to have 
```suggestion
        data.solidification && (material.solid.conductivity > 0.0 && material.solid.density > 0.0),
```

Reply:
For the case we don't have solidification we don't need the materal.solid parameters. Your assert would throw every time we don't have solidification enabled. We would need a both or nothing bool operator.

Reply:
Oh yes, I got it. Thank you for the explanation :-).